firebase-debug.log

```diff
@@ -0,0 +1,17 @@
+[debug] [2020-04-18T15:55:13.624Z] ----------------------------------------------------------------------
```

Comment:
I think we should just add this to the gitignore

Reply:
remove and add to gitignore